circles/models.py

```diff
@@ -2,6 +2,8 @@
 from django.db import models
 from users.models import User
 
+ME_CIRCLE = -2
+PUBLIC_CIRCLE = -1
```

Comment:
Depends. It's not so [nice](http://www.nice.org.uk/) that I cannot use these in the [template](https://github.com/7Pros/circuit/pull/182/files#diff-63d6d030a2c600c30247d13b745697feR20).